\

```diff
@@ -0,0 +1,30 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
Where / What is this file?

Reply:
sneaked in somehow. called ghostbusters, they blew it to hell.